algebird-core/src/main/scala/com/twitter/algebird/statistics/GaussianDistributionMonoid.scala

```diff
@@ -0,0 +1,28 @@
+package com.twitter.algebird.statistics
+
+import com.twitter.algebird.Monoid
+
+/**
+ * @mean - Mean
+ * @sigma - Standard deviation
+ * aka Normal distribution
+ */
+case class GaussianDistribution(mean: Double, sigma: Double)
```

Comment:
I actually have some code internal to Twitter that has this as well as some bootstrap and chernov/hoeffding-bound based randomized testing. I should release that (and perhaps make it work with the the code above rather than mine which is just a Reader monad on java.util.Random).